docx/oxml/header.py

```diff
@@ -0,0 +1,14 @@
+from . import OxmlElement
+from .xmlchemy import BaseOxmlElement, ZeroOrMore
+
+
+class CT_Hdr(BaseOxmlElement):
+    """
+    ``<w:hdr>``, the container element for the header content
+    """
+    p = ZeroOrMore('w:p', successors=())
+
+    @classmethod
+    def new(cls):
+        header_elm = OxmlElement('w:hdr')
+        return header_elm
```

Comment:
I included this in this commit just so we could talk about whether or not it's basically the right approach.. Feel free to nuke it. I have a feeling it might be a while before we get to this?